src/renderer/store/modules/settings.js

```diff
@@ -278,7 +278,8 @@ const state = {
   screenshotFolderPath: '',
   screenshotFilenamePattern: '%Y%M%D-%H%N%S',
   fetchSubscriptionsAutomatically: true,
-  settingsPassword: ''
+  settingsPassword: '',
+  allowDashAv1Formats: false
```

Comment:
Add trailing commas?
```suggestion
  allowDashAv1Formats: false,
```